src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -602,6 +775,11 @@ public virtual int LastIndexOf(Object value, int startIndex, int count)
 #if FEATURE_NETCORE
         [FriendAccessAllowed]
 #endif
+        /// <summary>Returns a read-only <see cref="T:System.Collections.IList" /> wrapper.</summary>
```

Comment:
This is happening because if the Attribute is inside a conditional markup, it is considered part of the leading trivia, different than what happens when it's a regular attribute. I'll probably have to test if inside the leading trivia, there's conditional markup and then insert the comment at a different position than just after the last leading trivia.